sql/flow.go

```diff
@@ -0,0 +1,129 @@
+package sql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/docker/docker/pkg/stdcopy"
+)
+
+func getPypiVersion() string {
+	url := "https://pypi.org/pypi/astro-sql-cli/json"
+	method := "GET"
+
+	httpClient := &http.Client{}
+	req, err := http.NewRequest(method, url, http.NoBody)
+	if err != nil {
+		fmt.Println(err)
+		return ""
+	}
+	res, err := httpClient.Do(req)
+	if err != nil {
+		fmt.Println(err)
+		return ""
+	}
+	defer res.Body.Close()
+
+	type response struct {
+		Info struct {
+			Version string `json:"version"`
+		} `json:"info"`
+	}
+	var resp response
+	err = json.NewDecoder(res.Body).Decode(&resp)
+	if err != nil {
+		fmt.Println(err)
+		return ""
+	}
+
+	return resp.Info.Version
+}
+
+func getContext(filePath string) io.Reader {
+	ctx, _ := archive.TarWithOptions(filePath, &archive.TarOptions{})
+	return ctx
+}
+
+func CommonDockerUtil(cmd []string, flags map[string]string) error {
+	ctx := context.Background()
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		panic(err)
```

Comment:
handle errors pls instead of panicking

Reply:
Hey @neel-astro, I have re-added panic'ing as I think there is nothing the caller can do to fix it.

Please let me know what you think i.e. why you think we should `return` the error.

cc @tatiana @pankajkoti

Reply:
@jemishp @kushalmalani I would be interested in what you think of using `panic` in this case.

Reply:
Can you return the error and handle gracefully?

Reply:
Instead of panicking, please return the `err` from `CommonDockerUtil()` so that when `executeCmd()` calls it; it can check for err and log it or return it back to the user for immediate feedback.